go/teams/storage/generic.go

```diff
@@ -219,8 +230,22 @@ func (s *memoryStorageGeneric) get(mctx libkb.MetaContext, teamID keybase1.TeamI
 	return state
 }
 
+func (s *memoryStorageGeneric) shutdown() {
+	s.Lock()
+	defer s.Unlock()
+	var err error
+	s.lru, err = lru.New(1)
```

Comment:
same deal, not sure if this is fine, but I can nil it out and make getLRU return an error instead